common/unsafe/src/test/java/org/apache/spark/unsafe/types/UTF8StringSuite.java

```diff
@@ -215,43 +215,6 @@ public void contains() {
     assertFalse(fromString("大千世界").contains(fromString("大千世界好")));
   }
 
-  @Test
-  public void containsInLowerCase() {
```

Comment:
Here and below, why are we removing these test cases?

Reply:
starting at https://issues.apache.org/jira/browse/SPARK-47410, we've separated collation-related UTF8String logic away from `UTF8String.java` and into `CollationSupport.java`

these changes for UTF8_BINARY_LCASE have nothing to do with the original UTF8String implementation, so all tests are now found only in CollationSupportSuite (instead of UTF8StringSuite) - we don't have a UTF8LCASEStringSuite, and I don't think we need one, so CollationSupportSuite should satisfy

Reply:
no need to run duplicate tests anyway, `CollationAwareUTF8String` is used only in `CollationSupport`